poetry.lock

```diff
@@ -102,7 +161,19 @@ files = [
     {file = "tomli-2.0.1.tar.gz", hash = "sha256:de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"},
 ]
 
+[[package]]
+name = "typing-extensions"
+version = "4.6.3"
+description = "Backported and Experimental Type Hints for Python 3.7+"
+category = "dev"
+optional = false
+python-versions = ">=3.7"
```

Comment:
For my own understanding, shouldn't this be `>=3.8` ?

Reply:
poetry.lock file is generated automatically. This python-versions tag indicates the requirement of the library regarding python version. Not a requirement for our package.